benchmark/Classic.fs

```diff
@@ -59,18 +59,14 @@ module ClassicHandler =
                 return err |> Error
         }
 
-    let setUrlBuilder<'r, 'err> (builder: UrlBuilder) (context: HttpContext) =
+    let withUrlBuilder<'r, 'err> (builder: UrlBuilder) (context: HttpContext) =
         Ok { context with Request = { context.Request with UrlBuilder = builder } } |> Task.FromResult
 
     let setUrl<'r, 'err> (url: string) (context: HttpContext) =
```

Comment:
Why is this still calles `setUrl` and not `withUrl`?

Reply:
It was fixed in a later PR: https://github.com/cognitedata/oryx/blob/f831ab209c557b978be81db079fc565d7192bfc3/src/Handler.fs#L73